frontend/src/metabase/collections/components/CollectionContent/constants.ts

```diff
@@ -1,6 +1,30 @@
 import type { CollectionItemModel } from "metabase-types/api";
 
 export const COLLECTION_PAGE_SIZE = 25;
+
+export const COLLECTION_CONTENT_COLUMNS = [
+  "type",
+  "name",
+  "lastEditedBy",
+  "lastEditedAt",
+  "actionMenu",
+] as const;
+
+export type CollectionContentTableColumn =
+  (typeof COLLECTION_CONTENT_COLUMNS)[number];
+
+export type CollectionContentTableColumnsMap = {
+  [key in CollectionContentTableColumn]: true;
```

Comment:
Is it possible to specify `false` as the key? (e.g. `{actionMenu: false}`) - just curious why the type requires `true` here

Reply:
@deniskaber Ah, I just realized this is an internal data structure - that makes total sense. My bad.